packages/environment-ember-loose/__tests__/intrinsics/each-in.test.ts

```diff
@@ -9,3 +9,38 @@ invokeBlock(eachIn({}, { a: 5, b: 3 }), {
     expectTypeOf(value).toEqualTypeOf<number>();
   },
 });
+
+// Can render inverse when undefined, null, or empty.
```

Comment:
Can you add a test against a maybe-undefined object (e.g. `{ a: number } | undefined`)? I'm not sure we'll infer the correct `key` and `value` types the way the default block's params are specified right now.